inception-project/inception
# #1066 - Recommender status info

**What's in the PR**
This PR includes websocket events and their distribution for scheduler tasks. The information on recommenders shown in the info-panel on the resommender-sidebar are updated via websocket.
No addition to the shown infos yet.

**How to test manually**
* configure a project with recommenders
* open the recommendation-sidebar
* you should be able to see that metrics are immediately updated after the recommenders are evaluated (e.g. look at the console output which shows the last f1-score).

**Automatic testing**

**Documentation**
